### ISSUE TITLE
Hyperparameters

### ISSUE THREAD

**Issue by JLC7** [joehoeller/Object-Detection-on-Thermal-Images#10]
When you trained the model from scratch did you use the hyperparameters in `train.py`?

```# Hyperparameters (results68: 59.2 mAP@0.5 yolov3-spp-416) https://github.com/ultralytics/yolov3/issues/310
hyp = {'giou': 3.31,  # giou loss gain
       'cls': 42.4,  # cls loss gain
       'cls_pw': 1.0,  # cls BCELoss positive_weight
       'obj': 52.0,  # obj loss gain (*=img_size/416 if img_size != 416)
       'obj_pw': 1.0,  # obj BCELoss positive_weight
       'iou_t': 0.213,  # iou training threshold
       'lr0': 0.00261,  # initial learning rate (SGD=1E-3, Adam=9E-5)
       'lrf': -4.,  # final LambdaLR learning rate = lr0 * (10 ** lrf)
       'momentum': 0.949,  # SGD momentum
       'weight_decay': 0.000489,  # optimizer weight decay
       'fl_gamma': 0.5,  # focal loss gamma
       'hsv_h': 0.0103,  # image HSV-Hue augmentation (fraction)
       'hsv_s': 0.691,  # image HSV-Saturation augmentation (fraction)
       'hsv_v': 0.433,  # image HSV-Value augmentation (fraction)
       'degrees': 1.43,  # image rotation (+/- deg)
       'translate': 0.0663,  # image translation (+/- fraction)
       'scale': 0.11,  # image scale (+/- gain)
       'shear': 0.384}  # image shear (+/- deg)

If not do you have your set of hyperparameters that you can share?

Thanks
